rootfs/scheduler/resources/pod.py

```diff
@@ -217,6 +217,16 @@ def _set_container(self, namespace, container_name, data, **kwargs):
                 "value": "1"
             })
 
+        # Inject MY_POD_IP variable with Pod's IP address
+        data["env"].append({
+            "name": "MY_POD_IP",
```

Comment:
Can you please rename it to `POD_IP` instead of `MY_POD_IP` and can you make it configurable only if `os.environ.get("DEIS_SHOW_POD_IP", True)` ?

Reply:
done!

Reply:
this doesn't seem to be working as expected. When I set `deis config:set DEIS_EXPOSE_POD_IP` nothing happens! I am guessing that it is because `DEIS_EXPOSE_POD_IP` needs to be set in the controller and not in the application. Am I correct? How do I do this?

Reply:
@Cryptophobia I did a refactor in a new commit so you can see the differences, and this one works perfectly:

```
$ deis releases -a doctor |grep EXPOSE
v276	2019-08-06T17:42:49Z	n0n0x added DEIS_EXPOSE_POD_IP

$ deis ps -a doctor
=== doctor Processes
--- web:
doctor-web-757b98b5b9-wx6fn up (v276)
doctor-web-757b98b5b9-xr2cb up (v276)

$ k doctor exec -i doctor-web-757b98b5b9-wx6fn -- env |grep POD_IP
WORKFLOW_RELEASE_SUMMARY=n0n0x added DEIS_EXPOSE_POD_IP
DEIS_EXPOSE_POD_IP=True
POD_IP=100.96.9.53
```

-------

```
$ deis releases -a doctor |grep EXPOSE
v277	2019-08-06T17:56:12Z	n0n0x deleted DEIS_EXPOSE_POD_IP
v276	2019-08-06T17:42:49Z	n0n0x added DEIS_EXPOSE_POD_IP

$ deis ps -a doctor
=== doctor Processes
--- web:
doctor-web-66787496ff-7fz24 up (v277)
doctor-web-66787496ff-ltgd7 up (v277)

$ k doctor exec -i doctor-web-66787496ff-7fz24 -- env |grep POD_IP
WORKFLOW_RELEASE_SUMMARY=n0n0x deleted DEIS_EXPOSE_POD_IP
```